core-java-modules/core-java-lang-3/src/test/java/com/baeldung/constantspatterns/ConstantPatternUnitTest.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.constantspatterns;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.Test;
+
+public class ConstantPatternUnitTest {
+    @Test
+    public void givenTwoNumbersAndAdd_whenCallingCalculatorOperatOneTwoNumbers_correctAnswerReturned() {
```

Comment:
```suggestion
    public void givenTwoNumbersAndAdd_whenCallingCalculatorOperateOnTwoNumbers_thenCorrectAnswerReturned() {
```

Reply:
Pushed a second commit to address this, since I had another change to make based on article feedback.